### PR TITLE
Use explode limit

### PR DESCRIPTION
**Pull request by mik-laj**
I am targetting this branch, because i wan't only fix code style

## Changelog

```markdown
### Changed
- Improve code style
```

### CODE REVIEW

**Comment by core23** [comment]
LGTM

File: Mailer/Mailer.php [hunk 1]
```diff
@@ -89,9 +89,9 @@ public function sendCommentNotification(CommentInterface $comment)
     protected function sendEmailMessage($renderedTemplate, $fromEmail, $toEmail)
     {
         // Render the email, use the first line as the subject, and the rest as the body
-        $renderedLines = explode("\n", trim($renderedTemplate));
-        $subject = $renderedLines[0];
-        $body = implode("\n", array_slice($renderedLines, 1));
+        $renderedParts = explode("\n", trim($renderedTemplate), 2);
+        $subject = $renderedParts[0];
+        $body = $renderedParts[1];
```
Comment (greg0ire) [inline Mailer/Mailer.php]: Maybe use `list()`?

Comment (mik-laj) [inline Mailer/Mailer.php]: Good point!

**Comment by OskarStark** [comment]
@Soullivaneuh stuck StyleCI build

**Comment by greg0ire** [comment]
LGTM

**Comment by greg0ire** [comment]
Thanks @mik-laj !

**Comment by OskarStark** [comment]
Thank you for your contribution @mik-laj

**Pull request merged**